057-AzureRouteServer/Coach/Solution-04.md

```diff
@@ -0,0 +1,20 @@
+# Challenge 04 - <Title of Challenge> - Coach's Guide 
```

Comment:
Is there no coach guide for challenge 4?

Reply:
@ravibak, any update?

Reply:
added.